src/Compiler.cpp

```diff
@@ -45,6 +48,15 @@ vector<Bytecode::Instruction>* convert(Token &token)
 	return data;
 };
 
+// Iterator, resets on true
+int iota(bool reset=false) 
```

Comment:
Use iota as a way to set the number of times an certain operation should be performed